src/DataFrames.jl

```diff
@@ -7,6 +7,7 @@ using Base.Sort, Base.Order, Base.Iterators
 using TableTraits, IteratorInterfaceExtensions
 import LinearAlgebra: norm
 using Markdown
+using PrettyTables
```

Comment:
do we want to reexport something (maybe it is not needed)

Reply:
Now that we can pass the arguments to `show`, the only point would be to reexport some variables defined by PrettyTables to change the format. For example, the table format. However, this is so low-level that maybe it is better to let the user uses `DataFrames.PrettyTables.<what they want>`.

Reply:
OK - so maybe just export `PrettyTables`?

Reply:
Done!

Reply:
Sorry to be late, but AFAICT this will only be needed in advanced cases, right? Otherwise people can just pass custom arguments to `show`? I'd say it's fine to require people to type `using PrettyTable` in these cases. That way we can keep the DataFrames API smaller.

Reply:
indeed the recommendation was when we assumed a different design. now it should be OK to pass arguments to `show` - is this correct @ronisbr?

Reply:
just to make sure - I understand that displaying is fully usable with this change without needing to load PrettyTables.jl? (i.e. most common options do not require custom types defined by PrettyTables.jl)

Reply:
Yes, unless the user wants to do things like highlighting cells.